src/shapes/Path.ts

```diff
@@ -92,6 +90,17 @@ export class Path<
     this.setBoundingBox(adjustPosition);
   }
 
+  /**
+   * This function is an helper for svg import. it returns the center of the object in the svg
+   * untransformed coordinates, by look at the polyline/polygon points.
+   * @private
+   * @return {Point} center point from element coordinates
+   */
+  _findCenterFromElement(): Point {
+    const bbox = this._calcBoundsFromPath();
+    return new Point(bbox.left + bbox.width / 2, bbox.top + bbox.height / 2);
+  }
```

Comment:
there is a definition in object too, but doesn't need to calculate the bbox and uses left, top, width height.

That implementation is not nice because it rely on the fact that for Rect, Circle, Ellipse and Text we apply left and top as values when parsing fromElement without caring for the origin.

That means that we position them wrongly, but then the function `removeTransformMatrixForSvgParsing` fixes it by calculating the center from those raw values and applying the correct position.

For path and polylines we do need to take care origin in consideration because somehow we want the points described in the polygon respected by the constructor. so if you  construct a polygon `[0,0], [0, 10], [10,10], [10, 0]` you expect to be in the top left corner of the canvas and the center in `[5,5]`.
This goes in conflict with the ability to detect where the polygon was in the SVG and this makes easier to keep the 2 problems divided.

separately, the investigation on fixing made possible to remove the fromSVG edge case and the strokeOffset ( that was there for svg only ) and at this point i want to get rid of them.

But i ll probalby split it in 2 prs since those are 2 different things